src/main/resources/org/jenkinsci/plugins/durabletask/powershellHelper.ps1

```diff
@@ -0,0 +1,85 @@
+﻿# By default PowerShell adds a byte order mark (BOM) to the beginning of a file when using Out-File with a unicode encoding such as UTF8.
```

Comment:
@batmat set this repo to commit as "unix style line ends" so you should probably make sure this file has binary attributes in git to make sure it has Windows line endings (if line endings matter - but the previous code was using `\r\n`)

Reply:
line endings do not seem to matter